README.md

```diff
@@ -182,14 +184,19 @@ If you have a build step for your code like [Babel](https://babeljs.io/) or [Typ
 
 ### prettier
 
-If you use [prettier](https://prettier.io/) to format your markdown, you may need to adjust your scripts to run prettier formatting after running this tool:
+If you use [prettier](https://prettier.io/) to format your markdown, you can provide a `postprocess` function to ensure the documentation generated by this tool is formatted correctly:
 
-```json
-{
-  "format": "prettier --write .",
-  "lint:eslint-docs": "npm run update:eslint-docs && git diff --exit-code",
-  "update:eslint-docs": "eslint-doc-generator && npm run format"
-}
```

Comment:
I think I'd like to keep this original script example--we can just move it after your new postprocess example. While the postprocess example is nice and efficient, in the real world, it might not be worth it for everyone:

* Greater complexity from having to write JS and add another config file
* Having to deal with breaking changes in Prettier's Node API
* In small repos, formatting time may be negligible
* The script example may be useful for combining eslint-doc-generator with other tools (not just prettier)